SPECS/kexec-tools/kdump.conf

```diff
@@ -183,3 +189,4 @@ core_collector makedumpfile -l --message-level 1 -d 31
 #dracut_args --omit-drivers "cfg80211 snd" --add-drivers "ext2 ext3"
 #fence_kdump_args -p 7410 -f auto -c 0 -i 10
 #fence_kdump_nodes node1 node2
+#disable_crash_dump_reboot 0
```

Comment:
please add new line at end of the file.  I found description here: https://stackoverflow.com/a/77201456/22141712